mantle/kola/tests/ignition/journaldEntry.go

```diff
@@ -29,6 +29,8 @@ func init() {
 		Name:        "coreos.ignition.journald-log",
 		Run:         sendJournaldLog,
 		ClusterSize: 1,
+		// Since RHCOS uses the 2x spec and not 3x.
```

Comment:
And just to emphasize this isn't about the Ignition spec version - we *could* backport your patch to the `spec2x` branch of Ignition upstream too and do a build for RHCOS.